servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -834,30 +844,114 @@ ServiceIdentity generateZTSServiceIdentity(com.yahoo.athenz.zms.ServiceIdentity
             }
             ztsService.setPublicKeys(ztsPublicKeys);
         }
-        
+
         return ztsService;
     }
-    
+
     String generateServiceIdentityName(String domain, String service) {
         return domain + "." + service;
     }
-    
+
     ServiceIdentity lookupServiceIdentity(DomainData domainData, String serviceName) {
-        
+
         List<com.yahoo.athenz.zms.ServiceIdentity> services = domainData.getServices();
         if (services == null) {
             return null;
         }
-        
+
         for (com.yahoo.athenz.zms.ServiceIdentity service : services) {
             if (service.getName().equalsIgnoreCase(serviceName)) {
                 return generateZTSServiceIdentity(service);
             }
         }
-        
+
         return null;
     }
-    
+
+    protected AthenzJWKConfig getAthenzJWKConfig(ResourceContext ctx) {
+        validateAthenzJWKConfigIsUpdated(ctx);
+        return athenzJWKConfig;
+    }
+
+    protected void validateAthenzJWKConfigIsUpdated(ResourceContext ctx) {
+        boolean rfc = false; // todo: configuration? something else?
```

Comment:
should we take it from system properties? what should be the default value?

Reply:
@havetisyan what about that?

Reply:
we should always use it as true. when we write the client part, at that we'll make sure we're handling the keys corre

Reply:
👍🏻